### ISSUE TITLE
Dual Fisheye Support?

### ISSUE THREAD

**Issue by petergerten** [yanwsh/videojs-panorama#48]
Is there any way to use this with a dual-fisheye video as input?
Example from a Ricoh Theta: https://developers.theta360.com/intl/common/img/dualfisheye.jpg

**Comment by yanwsh** [comment]
I think yes, Do you have video for dual fisheye? If you have, please send it to me, yanwsh@gmail.com

Thank you

**Comment by petergerten** [comment]
This is an example video: https://www.youtube.com/watch?v=Ty24agGJwCA
You can download it with one of those youtube-downloaders like https://www.onlinevideoconverter.com/video-converter

**Comment by petergerten** [comment]
Have you been able to try the video?

**Comment by yanwsh** [comment]
Oh, i didn't see your previous comment, will try this week.

**Comment by petergerten** [comment]
Any update on this?

**Comment by petergerten** [comment]
I tried some tweaks on the code myself but could not get it working. Anyone else got dual fisheye video to work?

**Comment by kevinyeo92** [comment]
any update on this? :)

**Comment by yanwsh** [comment]
still have issue with seamless.

**Comment by yanwsh** [comment]
<img width="900" alt="screen shot 2017-02-27 at 12 41 59 am" src="https://cloud.githubusercontent.com/assets/6547291/23350043/9a3d2d2c-fc85-11e6-87ea-2ae6f98f65cc.png">

**Comment by yanwsh** [comment]
You can see there has a line between two parts. This is because two fisheye have duplicate creative and i have trouble how to connect each other. Do you have any advice?

**Comment by kevinyeo92** [comment]
I am planning to find a way to convert dual fish eye into fish eye or equirectangular format

**Comment by yanwsh** [comment]
Yes, It's not difficult to render, the problem is how to merge them seamlessly is a big trouble.

**Comment by yanwsh** [comment]
That's the best i can do now.

http://yanwsh.x10host.com/demo/index_dualfisheye.html

Let me know if you find any documentation to help me solve the seamless issue. 

Thank you.

**Comment by petergerten** [comment]
That looks great already!
What part of the code do I have to change to get dual fisheye working in a custom setup?

Also, you might want to take a look at this for seamless rendering:
https://github.com/ricohapi/video-streaming-sample-app/blob/master/samples/common/thetaview.js

**Comment by kevinyeo92** [comment]
great job yanwsh.. really appreciate your great work 👍

**Comment by yanwsh** [comment]
@petergerten I checked the code which you provide, it works, but still not solve the seamless issue.

![screen shot 2017-02-27 at 11 13 59 pm](https://cloud.githubusercontent.com/assets/6547291/23391353/7e44498a-fd42-11e6-8608-5a3d0212ff69.png)

**Comment by yanwsh** [comment]
Hello, please download the latest version which support dual fisheye video. https://github.com/yanwsh/videojs-panorama/tree/master/dist

Let me know if the code work or not, might need specific configuration.

**Comment by hellaspanorama** [comment]
@yanwsh great job, I am trying to adopt the settings to play a dual fisheye front/back instead of up/down , but I cannot. Is it possible to modify a template according this video? http://hellaspanorama.com/test/xiaomi360/stream/dualfisheye.mp4
Thanks
